src/main/java/uk/gov/hmcts/reform/pip/account/management/model/PiUser.java

```diff
@@ -52,14 +53,15 @@ public class PiUser {
     /**
      * The user id of the user as per their provenance system.
      */
-    @NotNull
-    @NotBlank
+    @NotNull(message = "provenance user id must not be null")
+    @NotBlank(message = "provenance user id must not be blank")
     private String provenanceUserId;
 
     /**
      * Email of the user.
      */
-    @Email
+    @NotNull(message = "Email field must not be null")
```

Comment:
Could we remove the NotNull/Not blank here? As the checks for null/blank are already captured in the PiEmailConditionalValidator